server/src/main.cpp

```
@@ -110,107 +90,55 @@ int main(int argc, char *argv[])
-		if (it != bridges.end() && it->second->isDisconnected())
+		auto session = service.getSession(teamId);
+		if (session && session->isDisconnected())
 		{
 			ReplayEncoder::instance().setDeathReason(teamId, death_reason_t::DISCONNECTED);
 			ReplayEncoder::instance().setPlace(teamId, placeCounter--);
-		}
-		else if (it != bridges.end())
-		{
-			connectedTeamIds.push_back(teamId);
+			// Remove from active list
+			connectedTeamIds.erase(
```

Cppcheck (2.19.0)

[error] Calling 'erase' while iterating the container is invalid.

(invalidContainerLoop)